content/docs/iac/concepts/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -48,6 +83,10 @@ Moreover, because CDK depends on CloudFormation as the deployment engine, it sha
 
 Finally, while both CDK and Pulumi support automated testing, the scenarios they're able to support are quite different. For example, both CDK and Pulumi support unit testing, but Pulumi offers significant advantages as a result of the deep integration between language host and runtime. With Pulumi, you can run fast, in-memory (offline) unit tests that mock external calls to cloud providers, whereas with CDK, you're only able to run assertions against the rendered CloudFormation template synthesized by the CDK app, and there is no equivalent option for offline testing.
 
+## Pulumi vs. CDK: Better Together {#better-together}
+
+While there are differences and similiarities between Pulumi and AWS CDK, they can actually be used together for a more power experience in managing infrastructure. The pulumi-cdk library allows CDK code and constructs to be used directly in Pulumi programs, resulting in AWS resources being deployed and managed via Pulumi. Outputs of resources defined in a Pulumi program can be passed into AWS CDK constructs, and outputs from AWS CDK stacks can be used as inputs to other Pulumi resources. This allows CDK users to be able to use their existing constructs and access all the features of Pulumi Cloud (e.g., policy as code, drift detection, Insights and Copilot).
+
 The following table summarizes some additional similarities and differences between Pulumi and CDK, and the sections below the table go into more detail.
 
 ### Feature Comparisons
```

Comment:
Did you double check that all of this content is up to date, as good as it can be, and relevant?

Reply:
Yes, I wrote this one new. I based it off of the 1.0 CDK launch blog post.

Reply:
There is one outstanding question I have in the Questions channel about Policy as Code. I think they have released a few things that might give them some of these capabilities